Client.xcodeproj/project.pbxproj

```diff
@@ -293,6 +290,11 @@
 		2748626A25AD30A500BC40AA /* sqlcipher.xcframework in Copy Frameworks */ = {isa = PBXBuildFile; fileRef = 2748625C25AD308A00BC40AA /* sqlcipher.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		274AD90A278D455500FDF4D8 /* ManageWebsiteDataView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 274AD909278D455500FDF4D8 /* ManageWebsiteDataView.swift */; };
 		275034E326EF9E6A00CF4C8A /* WalletSettingsView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 275034E226EF9E6A00CF4C8A /* WalletSettingsView.swift */; };
+		2754EF5228175B6C00A94BAE /* BraveSharedTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2754EF5128175B6C00A94BAE /* BraveSharedTestUtils.swift */; };
+		2754EF5D28175BE500A94BAE /* BraveSharedTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2754EF5128175B6C00A94BAE /* BraveSharedTestUtils.swift */; };
+		2754EF5E28175BE600A94BAE /* BraveSharedTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2754EF5128175B6C00A94BAE /* BraveSharedTestUtils.swift */; };
+		2754EF5F28175BE600A94BAE /* BraveSharedTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2754EF5128175B6C00A94BAE /* BraveSharedTestUtils.swift */; };
+		2754EF6028175BE700A94BAE /* BraveSharedTestUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2754EF5128175B6C00A94BAE /* BraveSharedTestUtils.swift */; };
```

Comment:
Looks weird, multiple entries?

Reply:
They are shared testing util methods so they're added to all test targets (since you can't import test's from other tests and the test code can't be included in framework targets because it imports `XCTest`) which is why it looks weird :)